docs/ref/models/querysets.txt

```diff
@@ -979,6 +979,16 @@ database.
     performance, since you have done a database query that you haven't used. So
     use this feature with caution!
 
+.. note::
+
+    If you call the database-altering methods
+    :meth:`add()<django.db.models.fields.related.RelatedManager.add()>`,
```

Comment:
Same comments about link formatting as below.
Also need a versionchanged annotation here and in relations.txt as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features